client/components/LocalCompanies/BussinessCard.jsx

```diff
@@ -1,20 +1,24 @@
-import React, { useState } from 'react';
+import React, { useState, useContext } from 'react';
 import styled from 'styled-components';
 import Card from 'react-bootstrap/Card';
 import Badge from 'react-bootstrap/Badge';
 import StarRatings from 'react-star-ratings';
+import axios from 'axios';
+import { UserDataContext } from '../Data.jsx';
 
 
 const CardStyle = styled.div`
   display: inline-block;
   width: 18rem;
-  height: 30rem;
+  height: 550px;
   margin: 15px;
 `;
 
 const BussinessCard = ({ data }) => {
   //* * States * *//
   const [buttonColor, setButtonColor] = useState('secondary');
+  const { userData, setUserData } = useContext(UserDataContext);
+  console.log(userData.smoves.filter(smove => smove.isCurrentSmove)[0].favCompanies.filter(company => company.companyName !== data.name));
```

Comment:
did you mean to leave in this console log?

Reply:
fixed it